portal-bridge/src/bridge/beacon.rs

```diff
@@ -142,122 +162,127 @@ impl BeaconBridge {
     async fn serve_latest(
```

Comment:
This is correct,  we want to serve only historical summaries from a newly finalized beacon state (every ~ 32 slots).

Because it takes a few minutes at best to download the beacon state data from the consensus nodes, we don't want to check and download the next finalized state if the current download is still in progress.